src/pybind/mgr/rook/module.py

```diff
@@ -82,6 +87,24 @@ class RookOrchestrator(MgrModule, orchestrator.Orchestrator):
             default='local',
             desc='storage class name for LSO-discovered PVs',
         ),
+        Option(
+            'secure_monitoring_stack',
+            type='bool',
+            default=False,
+            desc='Enable TLS security for all the monitoring stack daemons'
+        ),
+        Option(
+            'prometheus_tls_secret_name',
+            type='str',
+            default='rook-ceph-prometheus-server-tls',
+            desc='name of tls secret in k8s for prometheus',
+        ),
+        Option(
+            'dashboard_tls_secret_name',
+            type='str',
+            default='rook-ceph-dashboard-server-tls',
+            desc='name of tls secret in k8s for dashboard',
+        )
```

Comment:
I think we can omit this variable if it's not used

Reply:
I have removed it